src/main/java/stsjorbsmod/cards/cull/deckoftrials/Blasphemer.java

```diff
@@ -0,0 +1,53 @@
+package stsjorbsmod.cards.cull.deckoftrials;
+
+import com.megacrit.cardcrawl.actions.common.ExhaustSpecificCardAction;
+import com.megacrit.cardcrawl.actions.common.MakeTempCardInHandAction;
+import com.megacrit.cardcrawl.cards.tempCards.Smite;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.cards.CustomJorbsModCard;
+import stsjorbsmod.characters.Cull;
+
+public class Blasphemer extends CustomJorbsModCard {
+    public static final String ID = JorbsMod.makeID(Blasphemer.class);
+
+    private static final CardRarity RARITY = CardRarity.SPECIAL;
+    private static final CardTarget TARGET = CardTarget.SELF;
+    private static final CardType TYPE = CardType.SKILL;
+    public static final CardColor COLOR = Cull.Enums.CULL_CARD_COLOR;
+
+    private static final int COST = -2;
+    private static final int SMITE_AMOUNT = 5;
+    private static final int UPGRADE_SMITE_AMOUNT = 1;
+
+    public Blasphemer() {
+        super(ID, COST, TYPE, COLOR, RARITY, TARGET);
+        this.exhaust = true;
+        this.cardsToPreview = new Smite();
+        magicNumber = baseMagicNumber = SMITE_AMOUNT;
+    }
+
+    @Override
+    public void triggerWhenDrawn() {
+        this.addToTop(new MakeTempCardInHandAction(new Smite(), this.magicNumber));
```

Comment:
Usually for a card, you want to use `addToBot` unless there's some specific exceptional reason to use `addToTop`. Actions that invoke other actions are usually the only place where `addToTop` is the more appropriate default choice.